tests/fmt.toml

```diff
@@ -1,9 +1,12 @@
+# This file can be used as a whole TOML file via rustfmt's --config-path option
+# The individual key-pairs can also used with the rustfmt --config option,
+# but requires removing the quotes to work
+
 comment_width=100
```

Comment:
Maybe add a comment saying all those options are unstable and require nightly channel